tests/integration/wait_times.yml

```diff
@@ -4,6 +4,7 @@ const.validate:
   ping: 5
   ping_long: 15
   vxlan_ping: 20
+  ping_xx_long: 40        # Needed in some VRF scenarios due to crazily slow control plane
```

Comment:
Not referenced in this PR?

Reply:
> Not referenced in this PR?

Nope, needed for the next PR (but I was doing multiple changes in parallel with running integration tests)